affiliated/affiliated_package_review_process.md

```diff
@@ -82,41 +82,46 @@ posted on the pull request and also in a reply to your email.
 
 ## Reviewing affiliated packages
 
-*These instructions are for coordinators and reviewers*
-
-One of the coordination committee members will be in charge of the review
-process (not necessarily the same person for each package). This coordination
-committee member (hereafter the *coordinator*) will find someone to carry out
-the main review of the package (hereafter the *reviewer*). In this sense the
-role  of the coordinator is similar to that of a journal editor.  However
-this is only partially true, because the coordinator (speaking for the
-coordination committee as a whole) may add their views to the review, unlike a
-journal editor who is often strictly impartial on the review itself.
-The pool of available reviewers will be anyone who has an official role on
-[the Astropy team](http://www.astropy.org/team.html).
-
-The coordinator sends out an email to possible reviewers one at a time to find
+*These instructions are for editors and reviewers*
+
+The review process will be overseen and coordinated by individuals that are in
+the *Affiliated package review editor* role (hereafter the *editors*). The
+editors will start off by finding someone to carry out the main review of the
+package (hereafter the *reviewer*). In this sense the role of the editor is
+similar to that of a journal editor (and hence the role name). However this is
+only partially true, because the editor may add their views to the review,
+unlike a journal editor who is often strictly impartial on the review itself.
+The pool of available reviewers will be anyone who has an official role on [the
+Astropy team](http://www.astropy.org/team.html) or anyone who has participated
+in the project enough to be familiar with the project guidelines and requirements.
+In some cases, it may be necessary to have multiple reviewers depending on the
+expertise required, and if scientific or algorithmic expertise is needed which
+is not available in the project, reviewers can be requested from outside the
+project.
+
+The editor sends out an email to possible reviewers one at a time to find
```

Comment:
The one at a time meant that if you need e.g. one reviewer, not to email five and then pick the first to reply. But that's perhaps too much detail. I've changed this to 'individually' to make it clear not to send blanket emails to 5 people at the same time.

How long the editors wait for reviewers to reply is up to them.

The reviewers are anonymous unless they say they are happy to not be. However, I think @eteq and I discussed this and thought it would make sense to have it the other way around - not anonymous unless requested by the reviewer. I will make that update in a separate PR though.